helpers/spark-submit.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import pwd
+import subprocess
+
+if __name__ == "__main__":
+    os.environ["HOME"] = pwd.getpwuid(os.getuid())[5]
+
+    if os.environ.get('SPARK_HOME') is None:
+        os.environ['SPARK_HOME'] = os.environ['SNAP']
+
+    submit_args = sys.argv[1:]
+    # if '--master' not in sys.argv:
+    #     default_kubeconfig = '{}/.kube/config'.format(os.environ["HOME"])
+    #     default_kubectl_cmd = '{}/kubectl'.format(os.environ['SNAP'])
+    #     default_master_kubectl_cmd = f"{default_kubectl_cmd} --kubeconfig {default_kubeconfig} config view -o jsonpath=\"{{.clusters[0]['cluster.server']}}\""
```

Comment:
1. context also should be passed here
2. it is very important to `--minify` here